drivers/include/ina3221.h

```diff
@@ -445,203 +437,251 @@ static inline int ina3221_enable_power_valid_alert(ina3221_t *dev,
  */
 static inline int ina3221_disable_timing_control_alert(ina3221_t *dev)
 {
-    return _ina3221_disable_alert(dev, INA3221_ALERT_TC);
+    return ina3221_disable_alert(dev, INA3221_ALERT_TC);
 }
 
 /**
- * @brief   Wrapper around @see _ina3221_disable_alert, to
+ * @brief   Wrapper around @see ina3221_disable_alert, to
  *          disable power valid alert
  *
  * @param[in,out] dev       Device handle
  *
- * @return @see _ina3221_disable_alert
+ * @return @see ina3221_disable_alert
  */
 static inline int ina3221_disable_power_valid_alert(ina3221_t *dev)
 {
-    return _ina3221_disable_alert(dev, INA3221_ALERT_PV);
+    return ina3221_disable_alert(dev, INA3221_ALERT_PV);
 }
 
 #endif /* MODULE_INA3221_ALERTS */
 
 /**
- * @brief Write configuration register value to configuration register
+ * @brief   Get enabled measurement channels from configuration value
  *
- * @param[in,out]   dev Device handle
- * @param[in]       cfg Configuration register value in host byte order
+ * @param[in]   cfg         Configuration value
  *
- * @return      INA3221_OK, on success
- * @return      -INA3221_I2C_ERROR, if I2C bus acquirement failed
- * @return      @see i2c_write_regs
+ * @return  Channel flags @see ina3221_channel_t
  */
-int _ina3221_set_config(ina3221_t *dev, uint16_t cfg);
+static inline uint16_t ina3221_config_get_enabled_channels(uint16_t cfg)
+{
+    return ((cfg & INA3221_ENABLE_CH1) ? INA3221_CH1 : 0) |
+           ((cfg & INA3221_ENABLE_CH2) ? INA3221_CH2 : 0) |
+           ((cfg & INA3221_ENABLE_CH3) ? INA3221_CH3 : 0);
+}
 
 /**
- * @brief Wrapper around @see _ina3221_set_config
+ * @brief   Enable measurement channels in configuration value
  *
- * @param[in, out]  dev Device handle
- * @param[in]       chs Channel enable flags @see ina3221_enable_ch_t
- * @param[in]       ns Number of samples
- * @param[in]       ctbadc Conversion time for bus voltage ADC
- * @param[in]       ctsadc Conversion time for shunt voltage ADC
- * @param[in]       mode Device operation mode
- *
- * @return      @see _ina3221_set_config
- */
-static inline int ina3221_set_config(ina3221_t *dev,
-                                     uint16_t chs,
-                                     ina3221_num_samples_t ns,
-                                     ina3221_conv_time_bus_adc_t ctbadc,
-                                     ina3221_conv_time_shunt_adc_t ctsadc,
-                                     ina3221_mode_t mode)
+ * @param[out]  cfg         Configuration value
+ * @param[in]   ch          Channel flags @see ina3221_channel_t
+ */
+static inline void ina3221_config_set_enabled_channels(uint16_t *cfg, uint16_t ch)
 {
-    return _ina3221_set_config(dev, chs | ns | ctbadc | ctsadc | mode);
+    if (cfg) {
+        uint16_t u16 = ((ch & INA3221_CH1) ? INA3221_ENABLE_CH1 : 0) |
+                       ((ch & INA3221_CH2) ? INA3221_ENABLE_CH2 : 0) |
+                       ((ch & INA3221_CH3) ? INA3221_ENABLE_CH3 : 0);
```

Comment:
```suggestion
        uint16_t u16 = ina3221_config_get_enabled_channels(ch);
```

Reply:
The new way is:
If you see `@see ina3221_channel_t` in the documenting of a `uint16_t` of a function then you pass
e.g. `INA3221_CH1 | INA3221_CH3`
The flags are translated to `INA3221_ENABLE_CH1 |  INA3221_ENABLE_CH3` in `ina3221_set_enable_channel()`
and  `INA3221_ENABLE_SUM_CH1 | INA3221_ENABLE_SUM_CH3` in `ina3221_set_enable_sum_channel`

Reply:
Oh, `ina3221_config_get_enabled_channels()` the positions of `INA3221_CH<X>` and `INA3221_ENABLE_CH<X>` are swapped compared to here.